vars/atomicHostTests.groovy

```diff
@@ -55,11 +55,28 @@ def call(body) {
             // Set Message Fields
             (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields('test.integration.complete')
             env.topic = topic
-
+            // Send message org.centos.prod.ci.pipeline.compose.test.integration.complete on fedmsg status = SUCCESS
+            messageUtils.sendMessage([topic:"${env.topic}",
+                                      provider:"${env.MSG_PROVIDER}",
+                                      msgType:'custom',
+                                      msgProps:messageProperties,
+                                      msgContent:messageContent])
+            env.MSG_PROPS = messageProperties
+            env.MSG_CONTENTS = messageContent
         }
     } catch (err) {
         echo "Error: Exception from " + current_stage + ":"
         echo err.getMessage()
+        (topic, messageProperties, messageContent) = pipelineUtils.setMessageFields('test.integration.complete')
```

Comment:
should the argument to setMessageFields be something like 'test.integration.failure' for the catch clauses?